ADVANCED-USAGE-README.md

```diff
@@ -0,0 +1,60 @@
+# PatternFly React
+This README covers advanced usage topics for PatternFly React users.  
+
+### Table of contents
+1. [Applying Red Hat Fonts](#Applying-Red-Hat-fonts)
+2. [Tree shaking](#Tree-shaking)
+
+## Applying Red Hat fonts
+If you would like to use Red Hat fonts instead of Overpass, simply add the class `.pf-m-redhat-font` to an element that wraps your application (ideally `<html>` or `<body>`) to adopt the CSS changes that introduce the Red Hat fonts into PatternFly.
```

Comment:
Since we are making this update  on the v4 branch, maybe we should just remove this section.  Red hat font will no longer be "opt-in" once we release this branch.  
cc @mcoker thoughts?

Reply:
That works for me. I suppose we should have a section that defines the new opt-in to overpass feature though. Do you think this would be a good place for that documentation? It's the same basic blurb, but the class is `.pf-m-overpass-font`